src/Grpc.Net.Client/GrpcChannelOptions.cs

```diff
@@ -287,6 +287,35 @@ public TimeSpan InitialReconnectBackoff
     /// </summary>
     public IServiceProvider? ServiceProvider { get; set; }
 
+    /// <summary>
+    /// Gets or sets the HTTP version to use when making gRPC calls.
+    /// <para>
+    /// When a <see cref="Version"/> is specified the value will be set on <see cref="HttpRequestMessage.Version"/>
+    /// as gRPC calls are made. Changing this property allows the HTTP version of gRPC calls to
+    /// be overridden.
+    /// </para>
+    /// <para>
+    /// A <c>null</c> value doesn't override the HTTP version of gRPC calls. Defaults to <c>2.0</c>.
+    /// </para>
+    /// </summary>
+    public Version? HttpVersion { get; set; }
```

Comment:
I assume something blows up downstream if someone sets `2.5` or `7.0`, but what happens if they select `1.1`?

Reply:
From the tests, it seems like this is handled somewhere (or might even be allowable?).

Reply:
1.1 will successfully create a connection if the server supports 1.1 and you're using gRPC-Web.